src/pages/stage-page/validate-stage-page.ts

```diff
@@ -1,16 +1,14 @@
 import { DevelopmentPlan, DevelopmentPlanTimetable } from "@lib/types/timetable";
-import Joi, { ValidationErrorItem } from "joi";
+import { ValidationErrorItem } from "joi";
 import { StagePageProps } from "./StagePage";
-import { eventDateSchema } from "../event-date-schema";
-
-const schema = Joi.object({
-  eventDate: eventDateSchema,
-  notes: Joi.string().allow(""),
-});
+import { eventSchema } from "../event-schema";
+import { StatusChangeEvent } from "@lib/constants";
 
 export const validateTimetableStage = (
   _developmentPlan: DevelopmentPlan,
   developmentPlanEvents: DevelopmentPlanTimetable[],
+  _statusChangeEvent: StatusChangeEvent | null,
+  _statusHasChanged: boolean | null, 
   formProps: StagePageProps
```

Comment:
Not a big thing but we know have 5 params, which is quite a few, we could switch to passing in an object and then functions could just destruct the fields they want

Reply:
done 🙂